frontend/src/component/changeRequest/ProjectChangeRequests/ChangeRequestsTabs/FeaturesCell.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, styled } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { VFC } from 'react';
+import { ConditionallyRender } from '../../../common/ConditionallyRender/ConditionallyRender';
+import { TooltipLink } from '../../../common/TooltipLink/TooltipLink';
+
+const StyledBox = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    width: '300px',
+    padding: theme.spacing(1, 0, 1, 2),
+}));
+
+const StyledLink = styled(Link)(({ theme }) => ({
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    textDecoration: 'none',
+    '&:hover, &:focus': {
+        textDecoration: 'underline',
+    },
+}));
+
+const StyledTooltipLink = styled(Link)(({ theme }) => ({
+    textDecoration: 'none',
+    '&:hover, &:focus': {
+        textDecoration: 'underline',
+    },
+}));
+
+const StyledTooltipContainer = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    fontSize: theme.fontSizes.smallBody,
+    width: '100%',
+    whiteSpace: 'nowrap',
+}));
+
+interface FeaturesCellProps {
+    value: any;
```

Comment:
can we give it a proper type even though the table may not have this information?